packages/components/src/components/drawer/drawer.stories.ts

```diff
@@ -0,0 +1,276 @@
+          }
+        ]
+      })}
+    </div>`;
+  }
+};
+
+export const Mouseless = {
+  parameters: { controls: { exclude: ['default', 'header', 'footer', 'contained', 'no-header'] } },
+  render: (args: any) => {
+    return html`<div class="mouseless" id="header-with-btn" style="width: auto; height: 95vh; position: relative;">
```

Comment:
There should be only one id called `header-with-btn`. You're using it twice at the moment when you're in the docs.